frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -63,9 +60,7 @@ export const ReportStreamHeader = () => {
             permissionCheck(PERMISSIONS.SENDER, authState) ||
             permissionCheck(PERMISSIONS.PRIME_ADMIN, authState)
         ) {
-            itemsMenu.splice(
-                1,
-                0,
+            itemsMenu.push(
```

Comment:
This might've slipped through. I thought Sean's PR caught it. Senders can still _see_ the nav item despite not having access to the Submissions page. If you move this down with the `PRIME_ADMIN` check below it should be fine 👍🏼

Reply:
Oh, good catch. Is this a merge issue?